libs/gi-localization/assets/locales/en/page_scanner.json

```diff
@@ -25,10 +25,6 @@
     "p1": "Scans all artifacts in your inventory. Has a manual scanning mode.",
```

Comment:
Can you add Adepti's new features with weapon scanning and enka-based character scanning?

Reply:
oh, right, that.